incubator/elasticsearch/README.md

```diff
@@ -0,0 +1,65 @@
+# Elasticsearch Helm Chart
+
+This image is using Fabric8's great work around the [kubernetes plugin](https://github.com/fabric8io/elasticsearch-cloud-kubernetes) for elasticsearch and their [image](https://hub.docker.com/r/fabric8/elasticsearch-k8s/) as parent.
+
+Credit to them!
+
+## Prerequisites Details
+
+* Kubernetes 1.3 with alpha APIs enable
+* PV dynamic provisioning support on the underlying infrastructure
```

Comment:
So elastic is all about the disk.  We need to put a note on here that with GCE you need to provision your own PVC that are ssd.